### ISSUE TITLE
.bin folder missing when using `--modules-folder`

### ISSUE THREAD

**Issue by kilianc** [yarnpkg/yarn#4297]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
```
ENV NODE_ENV production
ENV NODE_PATH /opt/app-node-modules
ENV PATH $PATH:/opt/app-node-modules/.bin

RUN yarn --no-emoji --modules-folder $NODE_PATH --pure-lockfile
```

`.bin` is nowhere to be found. 

**What is the expected behavior?**

Behavior (as per yarn@0.24.4 and node@7.10.0) should be `.bin` folder at `/opt/app-node-modules/.bin`

**Please mention your node.js, yarn and operating system version.**

* node@8.4.0
* yarn@0.27.5

P.S. I see this behavior bumping our docker images to 8.4.0 from 7.10.0

**Comment by BYK** [comment]
This looks like a duplicate of #3724. Please confirm :)

**Comment by schmunk42** [comment]
I have the same issue, looks like duplicate of #3724

**Comment by kilianc** [comment]
Any update on this, I have been stuck on `yarn@0.24.6` for months :(

**Comment by crgt5252** [comment]
Any update on this?  We're also blocked by this.

**Comment by EECOLOR** [comment]
@BYK yes, it's a duplicate of #3724

**Comment by BYK** [comment]
Closing in favor of #3724.
